en-us/multimc-auto-update.md

```diff
@@ -36,44 +36,35 @@ You'll have to download a new version of the pack that is specific to that Minec
-4. make them work in MultiMC
-
-*   Within the terminal, run another command:
-
-    `cd .minecraft && md5sum packwiz.json`
-
-    This will print "`randomnumber` - packwiz.json". Copy **only** the number.
-*   After this is done, open with a text editor the `pre-launch.sh` and on the 12th line where is says
-
-    `checksum=101c3431ead01d42ffbc80fecdccf903` replace the big number with the number you copied previously.
-
-    **NOTE:** You'll have to do this every time FO has an upgrade, otherwise the game will crash :(
-*   Now, head to line 20 of the same script
-
-    Here, you have to copy the mods' names you want to ignore from your `mods` folder and add them in order, like it is shown in the script. After each mods' name add a `\`, except from the last one, otherwise it won't work.
-* When a new _Minecraft_ release is out, the `pack.toml` has to be upgraded. So at the end of the line 5 there's a link, which somewhere in it should say a Minecraft version (eg. 1.18.2). So, whenever an update comes, you should edit that part of the link to match the MC version, otherwise [this](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/258) issue will be caused.
-* This is the final step! These scripts have to be run by MultiMC, and this is very simple:
-  1. Open MultiMC, right click your auto-upgrade instance and click "Edit Instance"
-  2. Go to Settings and click "Custom Commands" and enable them
-  3. Here, add to the "Pre-launch command" `/path/to/pre-launch.sh` (replace `/path/to/` with the location you saved the scripts) and to the "Post-exit command" `/path/to/post-exit.sh` (replace `/path/to/` with the location you saved the scripts)
-
-![Screenshot\_82](https://user-images.githubusercontent.com/77157639/157910323-02015782-7c9d-4a1c-a735-b5f0b75b79df.png)
+There is no official procedure for this yet, but RaptaG has made a tutorial and an improved version of the [Remty5's workaround](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/81). The user now only has to replace the instance parameters and list the mods that will be disabled. It has been confirmed to work on Linux, may also work for MacOS. Volunteers are free to port it to Windows as well.
+ 
+**Steps:**
+
+1. [Download and install jq](https://stedolan.github.io/jq/download/). This is a lightweight program that is needed for automatically adapting this script to any Minecraft version you use.
+2. Download the mod disabling scripts:
+   * Run the following command on a terminal - this will install the files and make them executable:
+`curl -Os https://raw.githubusercontent.com/RaptaG/fabulously-optimized/main/Packwiz/pre-launch.sh | curl -Os https://raw.githubusercontent.com/RaptaG/fabulously-optimized/main/Packwiz/post-exit.sh && chmod 755 pre-launch.sh post-exit.sh`
+   * You can install them wherever you like, just make sure to enter this place before, by running `cd /path/to/folder` (replace `/path/to/folder` with the folder's location).
```

Comment:
This part is a bit confusing - where _should_ the users install the scripts? Perhaps instruct them to install in the MultiMC instances folder, so that the scripts are accessible to all instances?

Reply:
Gimme a sec

Reply:
Not completely finished tho, missing some MacOS information

Reply:
Well, yeah, you added the macOS tutorial, but the last point still tells the user to "install wherever you like".

Reply:
> Well, yeah, you added the macOS tutorial, but the last point still tells the user to "install wherever you like". Let's just say the user should install it in the MultiMC's instances folder, make sure the tutorial matches that.

https://github.com/RaptaG/wiki/commit/3d995ba0e807e3e74693a966fd58b3fb0ecb55c5